CompuCell3D/core/Demos/experimental/PluginDemos/AdhesionFlexPython/error.txt

```diff
@@ -0,0 +1,16 @@
+Traceback (most recent call last):
```

Comment:
again this problem is related to wrong base class for the steppable. let's fix it by reverting to the demo version from current master branch